nixos/modules/services/web-apps/immich.nix

```
@@ -353,6 +354,21 @@ in
       };
     };
 
+    systemd.tmpfiles.settings = {
+      immich = {
+        # Redundant to the `UMask` service config setting on new installs, but installs made in
+        # early 24.11 created world-readable media storage by default, which is a privacy risk. This
+        # fixes those installs.
+        "${cfg.mediaLocation}" = {
+          d = {
```

```suggestion
          e = {
```
doesn't create the directory automatically. Otherwise we'd have to update the option description of `mediaLocation`.

---

Good catch, thanks!